vcd/resource_vcd_provider_vdc.go

```diff
@@ -0,0 +1,688 @@
+							Computed:    true,
+							Description: "True if compute capacity can grow or shrink based on demand",
+						},
+						"is_ha": {
+							Type:        schema.TypeBool,
+							Computed:    true,
+							Description: "True if compute capacity is highly available",
+						},
+					},
+				},
+			},
+			"compute_provider_scope": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Represents the compute fault domain for this Provider VDC. This value is a tenant-facing tag that is shown to tenants when viewing fault domains of the child Organization VDCs (for example, a VDC Group)",
+			},
+			"highest_supported_hardware_version": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The highest virtual hardware version supported by this Provider VDC",
+			},
+			"nsxt_manager_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "ID of the registered NSX-T Manager that backs networking operations for this Provider VDC",
+			},
+			"storage_container_ids": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Computed:    true,
+				Description: "Set of IDs of the vSphere datastores backing this provider VDC",
+			},
+			"external_network_ids": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Computed:    true,
+				Description: "Set of IDs of external networks",
+			},
+			"storage_profile_names": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Required:    true,
+				Description: "Set of storage profile names used to create this Provider VDC",
+			},
+			"storage_profile_ids": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Computed:    true,
+				Description: "Set of IDs to the storage profiles available to this Provider VDC",
+			},
+			"resource_pool_ids": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Required:    true,
+				Description: "Set of IDs of the resource pools backing this provider VDC",
+			},
+			"network_pool_ids": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional:    true,
+				Computed:    true,
+				Description: "Set IDs of the network pools used by this Provider VDC",
+			},
+			"universal_network_pool_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "ID of the universal network reference",
+			},
+			"host_ids": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Computed:    true,
+				Description: "Set with all the hosts which are connected to VC server",
+			},
+			"vcenter_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "ID of the vCenter server that provides the resource pools and datastores",
```

Comment:
Should we have `ForceNew: true` given that `update` cannot change the field

Reply:
Yes